Workbooks/Workloads/Enable monitoring/Enable monitoring.workbook

```diff
@@ -306,7 +306,7 @@
+              "query": "Operation\r\n| where Detail !contains \"mssql: Incorrect syntax near ','.\"\r\n| union (WorkloadDiagnosticLogs\r\n    | project Computer, OperationStatus=Status)\r\n| summarize Errors = countif(OperationStatus == 'Error'), All = count() by Computer\r\n| join kind=fullouter(\r\n    InsightsMetrics\r\n    | extend Tags = todynamic(Tags)\r\n    {HealthQuery}\r\n    | summarize by Computer, HasMetrics = true\r\n    | join kind=fullouter(InsightsMetrics\r\n    | extend Tags = todynamic(Tags)\r\n    | where Namespace == 'Computer' and Origin == 'solutions.azm.ms/wli'\r\n    | summarize (Time, WLI) = argmax(TimeGenerated, tostring(Tags.wli)) by Computer\r\n    | extend WLI_Major = extract('([0-9]+).[0-9]+.[0-9]+', 1, WLI), WLI_Minor = extract('[0-9]+.([0-9]+).[0-9]+', 1, WLI), WLI_Sub_Minor = extract('[0-9]+.[0-9]+.([0-9]+)', 1, WLI), WLI_Major_req = extract('([0-9]+).[0-9]+.[0-9]+', 1, '{WLIVersionRequired}'), WLI_Minor_req = extract('[0-9]+.([0-9]+).[0-9]+', 1, '{WLIVersionRequired}'), WLI_Sub_Minor_req = extract('[0-9]+.[0-9]+.([0-9]+)', 1, '{WLIVersionRequired}'))\r\n    on Computer\r\n    | extend Computer = iff(isnotempty(Computer), Computer, Computer1))\r\non Computer\r\n| summarize by Vm = iff(isnotempty(Computer), Computer, Computer1), Errors, All, HasMetrics, Status = iff(HasMetrics or isnotempty(HasMetrics), iff(Errors < 1 or isempty(Errors), 'Collecting', 'Collecting with errors'), 'Not Collecting'), LogsFilter = iff(Errors < 1 or isempty(Errors), \"All Logs\", \"Errors\"), WLI, Action = iff(todouble(WLI_Major) >= todouble(WLI_Major_req) and todouble(WLI_Minor) >= todouble(WLI_Minor_req) and todouble(WLI_Sub_Minor) >= todouble(WLI_Sub_Minor_req), 'Configure', 'Update')",
```

Comment:
main change is here union foor WorkloadDiagnosticLogs, and joining on Computer instead of _ResourceId. I think we might run into a problem if there are same computer names from different subs, since _ResourceId is more unique. But the WorkloadDiagnosticLogs doesn't have a _ResourceId column